checkstyle/checks/FileLengthCheck.hx

```diff
@@ -0,0 +1,17 @@
+package checkstyle.checks;
+
+import checkstyle.LintMessage.SeverityLevel;
+import haxeparser.Data.Token;
+
+@name("FileLength")
+@desc("Max number of lines per file (default 1000)")
+class FileLengthCheck extends Check {
+
+	public var severity:String = "WARNING";
+
+	public var maxLength:Int = 1000;
```

Comment:
Can you please change `maxLength` to `max` similar to http://checkstyle.sourceforge.net/config_sizes.html